entry_point_test.go

```diff
@@ -52,3 +54,7 @@ func TestRunObservabilitySuiteSingle(t *testing.T) {
 func TestRunInterdomainSuite(t *testing.T) {
 	suite.Run(t, new(interdomain.Suite))
 }
+
+func TestRunRvlanSuite(t *testing.T) {
```

Comment:
At this moment for running interdomain and single cluster tests separately we're using `Interdomain` and `Single` based regexp: 
https://github.com/networkservicemesh/integration-k8s-kind/blob/main/.github/workflows/ci.yaml#L109
https://github.com/networkservicemesh/integration-k8s-kind/blob/main/.github/workflows/ci.yaml#L157|
 
```suggestion
func TestRunRvlanSuiteSingle(t *testing.T) {
```

Reply:
OK. Updated. Thanks for clarifying this @denis-tingaikin